e2e/tests/functional/plugins/conditionSet/conditionSet.e2e.spec.js

```diff
@@ -475,3 +475,110 @@ test.describe('Basic Condition Set Use', () => {
     });
   });
 });
+
+test.describe('Condition Set Composition', () => {
+  let conditionSet;
+  let exampleTelemetry;
+
+  test.beforeEach(async ({ page }) => {
+    await page.goto('./', { waitUntil: 'domcontentloaded' });
+
+    // Create Condition Set
+    conditionSet = await createDomainObjectWithDefaults(page, {
+      type: 'Condition Set',
+      name: 'Condition Set with Telemetry and Conditions'
```

Comment:
optional: we can remove the `name` and it will generate a random uuuid name. then just reference conditionSet.name